.github/workflows/add_to_project.yaml

```diff
@@ -0,0 +1,16 @@
+name: Add issues to GH project
```

Comment:
Don't we have license headers in place for this project?

Reply:
It doesn't seem to be the case, looking at the other workflows.
The `license-check` workflow file is about Fossa scanning, which is slightly different.